testbed/hil_constants.py

```diff
@@ -20,4 +20,6 @@ class VehicleBoardId(Enum):
 
 
 class HilBoardId(Enum):
-    VCU_HIL = 2
\ No newline at end of file
+    VCU_HIL = 2
+    
+    PDU_HIL = 3
```

Comment:
The red circle with the line means that there is no newline at the end of the file which is standard practice. Suggest adding one in